docs/oob-s3-ring.md

```diff
@@ -0,0 +1,165 @@
+# Out-of-band (OOB) Updates from RING to Zenko
+
+## Description
+
+This design document describes the feature of ingesting metadata of objects put
+in RING over S3 Connector into Zenko.
+
+The development involves writing a service in Backbeat that allows
+ingesting existing objects as well as newly created objects from a remote
+S3C/RING into Zenko.
+
+## Purpose
+
+OOB updates from RING feature allows Zenko features on the configured RING
```

Comment:
maybe it's just me nitpicking, but I feel like it should be stressed that the OOB feature will allow for Zenko features to work on objects that have been put or will be `PUT` outside of Zenko rather than the RING bucket as a whole.